index.html

```diff
@@ -38,7 +38,7 @@ <h3>las2peer-frontend-statusbar demo with slots</h3>
                                          baseurl="https://las2peer.dbis.rwth-aachen.de:9098"
                                          oidcpopupsigninurl="http://localhost:8081/callbacks/popup-signin-callback.html"
                                          oidcpopupsignouturl="http://localhost:8081/callbacks/popup-signout-callback.html"
-                                         oidcsilentsigninturl="http://localhost:8081/callbacks/silent-callback.html"
```

Comment:
Just a quick question..why do we go with a combination of a las2peer.dbis base url and a localhost oidc?

Reply:
To be honest, I don't know. Going only with localhost would make more sense as index.html is only used for the demo (and the default value of baseurl is also http://127.0.0.1:8081).

Reply:
Changed the baseurl as mentioned and updated package-lock with the newest user-widget version